### PR TITLE
[Stratconn 3718] Add Remove Profile

### PR DESCRIPTION
**Pull request by harsh-joshi99**
Add Remove Profile action to klaviyo which works with rETL and mimics Remove Profile from List (Engage).

JIRA -> [STRATCONN-3718](https://segment.atlassian.net/browse/STRATCONN-3718)

## Testing

Test Completed Successfully in staging.
Screenshot 2024-04-16 at 11 26 56 AM

**RETL**
Mapping
Screenshot 2024-04-16 at 11 53 18 AM

Screenshot 2024-04-16 at 11 53 27 AM

Result
Screenshot 2024-04-16 at 11 52 38 AM

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

[STRATCONN-3718]: https://segment.atlassian.net/browse/STRATCONN-3718?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/destination-actions/src/destinations/klaviyo/removeProfile/index.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { ActionDefinition, DynamicFieldResponse, PayloadValidationError } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import { Payload } from './generated-types'
+
+import { getListIdDynamicData, getProfiles, removeProfileFromList } from '../functions'
+import { enable_batching } from '../properties'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Remove Profile',
+  description: 'Remove profile from list',
```
Comment (AnkitSegment) [inline packages/destination-actions/src/destinations/klaviyo/removeProfile/index.ts:10]: Please change the description to look different from RemoveProfileFromList Actions.

**Review by AnkitSegment — DISMISSED** [review]
LGTM

**Review by varadarajan-tw — COMMENTED** [review]
Could you merge latest changes from master to your branch? Couple of tests are failing.

Otherwise, the PR LGTM ✨

File: packages/destination-actions/src/destinations/klaviyo/removeProfile/index.ts (hunk 1 above)
Comment (harsh-joshi99) [reply packages/destination-actions/src/destinations/klaviyo/removeProfile/index.ts:10]: The action is exactly same as Remove profile from list and is added to support RETL. I think same description should be ok?

**Review by varadarajan-tw — APPROVED** [review]

**Pull request merged**